src/shim/clock.rs

```diff
@@ -27,4 +27,12 @@ mod tests {
             fvm_shared3::clock::EPOCH_DURATION_SECONDS
         )
     }
+
+    #[test]
+    fn fvm4_shim_of_const_epoch_duration_seconds() {
+        assert_eq!(
+            super::EPOCH_DURATION_SECONDS,
+            fvm_shared4::clock::EPOCH_DURATION_SECONDS
+        )
+    }
```

Comment:
This can be a static assertion instead of a test.

Reply:
Do you mean using `const_assert_eq!`? Where should I put the assertion code?

Reply:
I removed our own re-definition of `EPOCH_DURATION_SECONDS` and just re-exported it from the FVM4 + removed the tests.